double arm1;
double arm2;
double arm3;
double arm4;
double arm5;
double gripper_l_param;
double gripper_r_param;
